docker-compose.yml

```diff
@@ -286,3 +325,8 @@ volumes:
   elasticsearch-data:
   quickstatements-data:
   portainer-data: # volume to save settings of portainer
+  prometheus_data:
+  grafana_data:
+
+networks:
+    wdqs:
```

Comment:
Why was this added?

Reply:
same as above, residues I think, stick with main